src/pages/ecosystem/EcosystemChekout.vue

```diff
@@ -0,0 +1,192 @@
+
+const plan = Object.freeze({
+  ECOSYSTEM: 'ECOSYSTEM',
+  CHILD: 'CHILD'
+})
+
+export default {
+  name: 'page-ecosystem-checkout',
+  components: {
+    WalletHypha: () => import('~/components/profiles/wallet-hypha.vue'),
+    Widget: () => import('~/components/common/widget.vue')
+  },
+
+  data () {
+    return {
+      balances: [],
+      state: 'WAITING',
+      plan
+    }
+  },
+
+  computed: {
+    ...mapGetters('accounts', ['account', 'isAdmin']),
+    ...mapGetters('dao', ['configs', 'dho', 'selectedDao', 'selectedDaoPlan']),
+
+    loading () { return !this.dho },
+    type () { return this.$route.params.type }, // DAO, ECOSYSTEM,
+    quantity () { return this.$route.params.quantity || 1 },
+
+    daoId () { return this.$route.params.daoId },
+    daoName () { return this.$route.params.daoName },
+
+    selectedPlan () { return this.type ? this.PLANS.find(_ => _.type === this.type) : {} },
+    usdPerHypha () { return this.configs && this.configs.usdPerHypha ? this.configs.usdPerHypha : 1 },
+
+    PLANS () {
+      if (!this.dho) return []
+      return [
+        {
+          type: plan.ECOSYSTEM,
+          name: 'Anchor DAO',
+          priceUSD: Number(this.dho.ecosystemPrice.replace(' USD', '')) + Number(this.dho.ecosystemPriceStaked.replace(' USD', '')),
+          priceHypha: Math.ceil((Number(this.dho.ecosystemPrice.replace(' USD', '')) + Number(this.dho.ecosystemPriceStaked.replace(' USD', ''))) / this.usdPerHypha),
+          stakedUSD: Number(this.dho.ecosystemPriceStaked.replace(' USD', '')),
+          stakedHypha: Math.ceil(Number(this.dho.ecosystemPriceStaked.replace(' USD', '')) / this.usdPerHypha)
+        },
+        {
+          type: plan.CHILD,
+          name: 'Each Additional DAO',
+          priceUSD: Number(this.dho.ecosystemChildPrice.replace(' USD', '')) + Number(this.dho.ecosystemChildPriceStaked.replace(' USD', '')),
+          priceHypha: Math.ceil((Number(this.dho.ecosystemChildPrice.replace(' USD', '')) + Number(this.dho.ecosystemChildPriceStaked.replace(' USD', ''))) / this.usdPerHypha),
+          stakedUSD: Number(this.dho.ecosystemChildPriceStaked.replace(' USD', '')),
+          stakedHypha: Math.ceil(Number(this.dho.ecosystemChildPriceStaked.replace(' USD', '')) / this.usdPerHypha)
+        }
+      ]
+    }
+
+  },
+
+  methods: {
+    ...mapActions('dao', ['activateDAOPlan', 'activateEcosystem', 'activateChildDao']),
+    ...mapActions('profiles', ['getHyphaBalance']),
+
+    formatMoney (amount) { return amount ? new Intl.NumberFormat().format(parseInt(amount), { style: 'currency' }) : 0 },
+    async fetchHyphaBalance (account) {
+      if (!account) return
+      try {
+        const hyphaBalance = await this.getHyphaBalance(account)
+        this.balances = [
+          { ...hyphaBalance, icon: 'QmQoxvKHRuNknRF4A445vJKAPZvrH5fVTo6N4Zyn1naEKn:png' }
+        ]
+      } catch (error) {
+      }
+    },
+
+    async onActivateAnchorDao () {
+      try {
+        this.state = 'ACTIVATING'
```

Comment:
@arsenijesavic Constants over strings, you are using `ACTIVATING` more then once